project/dagger-query.sh

```diff
@@ -0,0 +1,3 @@
+export PATH="$PATH:$HOME/bin"
+
+bazel run --sandbox_debug --sandbox_block_path=/usr/local //src/com/google/daggerquery/executor:query_executor $@
```

Comment:
Should `//src/com/google/daggerquery/executor:query_executor` be changed to reference user's `dagger_query` macro now?
If this is specified as the first parameter of the user input, then the `$@` will have it and we can simply remove `//src/com/google/daggerquery/executor:query_executor`.

Reply:
Please also update the `README.md` instructions afterwards.

Reply:
Done 💫 [I've put these instructions](https://github.com/googleinterns/dagger-query/pull/23/commits/add4e702142aba60fdb59aa86418f414a3773754) in the third step in a README.